algorithms/src/snark/marlin/ahp/prover/state.rs

```diff
@@ -114,13 +114,13 @@ impl<'a, F: PrimeField, MM: MarlinMode> State<'a, F, MM> {
 
                 let first_padded_public_inputs = &variable_assignments[0].0;
                 let input_domain = EvaluationDomain::new(first_padded_public_inputs.len()).unwrap();
-                let batch_size = variable_assignments.len();
+                let batch_size = u32::try_from(variable_assignments.len())?;
```

Comment:
Same, let's change this to `try_into()?`

Reply:
5ebe7b258